### PR TITLE
[ci] Move CI to Github Actions

### LINKED ISSUE #131: [CI] Move to github actions.
Move CI from travis to github actions.

### PR DESCRIPTION
**Pull request by ManishShah120**
Closes #131

### CODE REVIEW

**Comment by ManishShah120** [comment]
@devkapilbansal this is working :+1:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        run: pip install -U "pip==20.2.4" wheel
+
+      - name: Install test dependencies
+        run: |
+          pip install -U -r requirements.txt
+          pip install pylinkvalidator
+
+      - name: QA checks
+        run: |
+          ./run-qa-checks
```
Comment (devkapilbansal) [inline .github/workflows/ci.yml:40]:
```suggestion
        run: ./run-qa-checks
```

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@ManishShah120 just one minor change :point_down:

**Review by devkapilbansal — APPROVED** [review]
LGTM 👍

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**